lib/chef/knife/ec_base.rb

```diff
@@ -121,6 +127,33 @@ def rest
         @rest ||= Chef::ServerAPI.new(server.root_url, {:api_version => "0"})
       end
 
+      def remote_users
+        @remote_users ||= rest.get('/users')
+      end
+
+      def remote_user_list
+        @remote_user_list ||= remote_users.keys.map(&:to_sym)
+      end
+
+      def local_user_list
+        @local_user_list ||= Dir.glob("#{@dest_dir}/users/*\.json").map { |u| File.basename(u, '.json').to_sym }
+      end
+
+      def for_each_user_purge
+        purge_list = if opt_parser.default_argv[1] == 'backup'
```

Comment:
I moved the implementation to each backup|restore class as well as in base where it raises an `Chef::Knife::EcBase::UnImplemented` error.